### DISCUSSION TITLE
View details for events from read-only calendars

### DISCUSSION THREAD

**Discussion by jeremytoce** [davish/obsidian-full-calendar#370] [Feature Requests]
Whenever I click on a item in anyt view, I receive this error message. My user intuition tells me it is intended to open up a new note. Unsure if it's a bug or user error!

**Comment by pcause** [comment]
I get this as well.  I am clocking on items for an online calendar.  Certainly an issue with handling remote calendars that didn't exist previously

**Comment by dojoteef** [comment]
This isn't a bug. Remote calendars do not support being edited currently. Follow #25 to see progress on this. I have a PR #96 that takes a step towards two-way sync. In that PR, when you click on the remote calendar event it will pop up the edit event modal dialog, but doesn't allow editing.

@davish I'd recommend closing this out as it will be addressed with my PR.

**Comment by pcause** [comment]
This wasn't my use case, as I just wanted to display the details, not edit.  In the case where there is no editing, at least the event details should be shown, which is what I wanted.  I don't think 2 way sync is needed for view event without editing.

**Comment by dojoteef** [comment]
@pcause then when my PR gets merged, it should address your issue of being able to view the event details. I guess for @jeremytoce, it won't open a new note until there is two-way sync as in #25.

**Comment by pcause** [comment]
@dojoteef , thanks.  The calendar I was using is not a Caldav calendar.  Assuming your PR will handle this as well.

**Comment by dojoteef** [comment]
> @dojoteef , thanks. The calendar I was using is not a Caldav calendar. Assuming your PR will handle this as well.

You previously stated:

> Certainly an issue with handling remote calendars that didn't exist previously

Are you using a remote .ics calendar then?

**Comment by davish** [comment]
I can confirm that both CalDAV and ics calendars are both read-only currently. I'm renaming this issue and recategorizing it as an enhancement to reflect that viewing info for readonly events is a new feature.

@dojoteef we can close out this issue when your next PR gets in to `main` :)

**Reply by planefun2** [reply]
did this ever get merged? I still have this issue in 0.10.7

**Comment by jordie23** [comment]
Migrating from "Reflect" this is one thing I miss. My Google Calendar is read-only, that's fine. But I want to be able to view the events, right click on them and have a new note or a new section in daily note added with info about the event. It doesn't need to sync anything back to the source calendar, I just want to make it quick and easy to start note taking in a meeting.

Is there a current PR or bit of around this at the moment?

**Reply by walkabout21** [reply]
@jordie23 this is my use case as well.  I have a "meetings" folder and I want to generate a note from a remote calendar entry with the name, subject, date time, participants as frontmatter and file name as meeting Name - date  so I can take notes quickly.  I don't really need this linked back to the calendar/synced.  Just needs to be easy to find in my vault for reference.

**Reply by balazs-z-horvath** [reply]
I have the exact same use-case, I'm looking for a quick and easy way to start meeting notes and to pull meeting details into my notes. 
I previously used Agenda for this, but trying to get all my note-taking into Obsidian.

**Comment by YouFoundJK** [comment]
Please do check out this [discussion](https://github.com/YouFoundJK/plugin-full-calendar/discussions/102) and let me know your feedback on our developments.
